core/Enclave/enclave_hsm.cpp

```diff
@@ -225,12 +248,25 @@ static sgx_status_t sgx_gcm_encrypt(const sgx_aes_gcm_128bit_key_t *key,
  * sgx_rijndael128GCM_decrypt() is running in enclave
  * 
  */
-static sgx_status_t sgx_gcm_decrypt(/* param */)
+static sgx_status_t ehsm_gcm_decrypt(const sgx_aes_gcm_128bit_key_t *key,
+                                    uint32_t *plaintext_size, uint8_t *plaintext,
```

Comment:
[Why is plaintext_size defined as a pointer? Unless the caller does not know the size of plaintext, and have to call the function at the 1st time to get the size.  Otherwise, no need to define it as a pointer.

Reply:
Except for the function name, I haven't made any changes. The previous definition is like this. I also tend to modify it

Reply:
The previous usage needs to call the function twice.  The 1st time calling is to get the size of the plaintext